### PR TITLE
Adding an EFT Secure Icon, update Masterpass

### PR DESCRIPTION
**Pull request by KevinDunbar93**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

File: db/payment_icons.yml [hunk 1]
```diff
@@ -402,7 +402,11 @@
   name: vipps
   label: Vipps
   group: other
-  
+-  
   name: enets
   label: eNETS
   group: other
+
```
Comment (maximevaillancourt) [inline db/payment_icons.yml]: This line is missing a `-` to denote a new entry (see [example 2.4 from the YAML docs](https://yaml.org/spec/1.2/spec.html#id2760193))

**Comment by KevinDunbar93** [comment]
@NathanPJF. I have made the requested changes.

**Comment by NathanPJF** [comment]
Thanks, @KevinDunbar93 

Are good to delete this Masterpass related PR? https://github.com/activemerchant/payment_icons/pull/153

**Comment by tejasmi5** [comment]
@NathanPJF we made this pull request a few months ago but we're not seeing the option to select these payment methods in the gateway settings. Is this something you could help with?

**Comment by tejasmi5** [comment]
Hi @NathanPJF and @maximevaillancourt , just following up here. We're not seeing EFT Secure as an option in the gateway settings for payment methods. Is there something else we need to do after this pull requested was merged?

**Comment by tauthomas01** [comment]
Hi @tejasmi5, in which platform you don't see the icons?

**Comment by tejasmi5** [comment]
Hi @tauthomas01, weirdly enough I do see it now, but the logo doesn't look right which makes me think this it didn't come from this PR. 

The logos are part of the gateway settings in the partner dashboard in Shopify. Here's a screenshot - 
image

**Comment by NathanPJF** [comment]
@tejasmi5 Could you clarify what doesn't look right? In your screenshot I can't see of any obvious issues, aside from some logo alignment. Perhaps you aren't referring to the Mastercard logo. That appears to be the same as `master.svg` https://github.com/activemerchant/payment_icons/blob/84912a44e51be241dbe41571d4385434dc65c47b/app/assets/images/payment_icons/master.svg

To set proper expectations: What shows up in Shopify's Partner Dashboard is out of scope of this `payment_icons` repo. The purpose of the repo is to be a repository of payment icons. Your best course of action may be to reach out to a Shopify Partner manager.

**Pull request merged**